kentik_api_library/tests/e2e/synthetics/test_flow.py

```diff
@@ -1,24 +1,27 @@
+from copy import deepcopy
+
 import pytest
 
 from kentik_api.synthetics.synth_tests import FlowTest
 from kentik_api.synthetics.synth_tests.base import PingTask, TraceTask
 from kentik_api.synthetics.synth_tests.flow import FlowTestSettings, FlowTestSpecific
 from kentik_api.synthetics.types import DirectionType, FlowTestSubType, IPFamily, Protocol, TestStatus, TestType
 
-from .utils import HEALTH, client, credentials_missing_str, credentials_present, pick_agent_ids
+from .utils import HEALTH1, HEALTH2, client, credentials_missing_str, credentials_present, pick_agent_ids
 
 
 @pytest.mark.skipif(not credentials_present, reason=credentials_missing_str)
 def test_flow_crud() -> None:
-    settings = FlowTestSettings(
+    agents = pick_agent_ids(count=2)
+    settings1 = FlowTestSettings(
         family=IPFamily.V4,
         period=60,
-        agent_ids=pick_agent_ids(),
-        health_settings=HEALTH,
-        ping=PingTask(timeout=3000, count=5, delay=200, protocol=Protocol.ICMP, port=2222),
+        agent_ids=[agents[0]],
+        health_settings=HEALTH1,
+        ping=PingTask(timeout=3000, count=5, delay=200, protocol=Protocol.ICMP),
         trace=TraceTask(timeout=22500, count=3, limit=30, delay=20, protocol=Protocol.UDP, port=3343),
         flow=FlowTestSpecific(
-            target="123",
+            target="www.example.com",
```

Comment:
Since you set `type=FlowTestSubType.CITY` the `target` should be a name of a city.

Reply:
Updating